### DISCUSSION TITLE
Email is tied to school after applying?

### DISCUSSION THREAD

**Discussion by Papierkorb2292** [community/community#58252] [GitHub Education]
### Hi everyone,

I wanted to apply for the student developer pack and, like the form says, I selected my email address that I'm using for school, which is not issued by my school.
However, after my application was rejected and I went to reapply, I noticed that now selecting my email also selects the school I previously selected and I can't change what school is selected now. I also get the notification that emails ending like mine need to upload additional proof.
This isn't an issue for me applying at the moment, but my email should not be directly connected to my school (what happens if one uses a googlemail address for example?), how can I change that?

**Comment by the-officialjosh** [comment]
If you have encountered issues with the student developer pack application regarding the email address selection and school association, it is recommended to reach out to the support team or contact the relevant support channels provided by the organization offering the student developer pack.

Explain the situation and express your concern about the email address being directly connected to your school, even if it is not issued by your school. They should be able to guide you on how to update or modify the school association or provide further instructions on the additional proof required for email addresses like yours.

Remember to provide specific details about your situation and be clear in your communication with the support team to ensure a prompt and accurate response.
